packages/visual-dev/src/components/FormTemplates.tsx

```diff
@@ -133,15 +126,19 @@ function Label(props: LabelProps) {
   }
   return (
     <div style={{ marginTop: "40px", marginBottom: "10px" }}>
-      <label className="control-label" htmlFor={id} style={{marginBottom:"0px"}}>
+      <label
+        className="control-label"
+        htmlFor={id}
+        style={{ marginBottom: "0px" }}
+      >
```

Comment:
could make this a styled component and hunt down the `.control-label` css if there is any, probably not necessary because it's only used once here though.